apps/backend/src/models/proxy_config.py

```
@@ -0,0 +1,264 @@
+from sqlalchemy.orm import relationship, validates
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.sql import func
+import uuid
+
+from apps.backend.src.core.database import Base
+
+
+class ProxyConfig(Base):
+    """
+    Main table for reverse proxy configurations
+    
```

Ruff (0.12.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)